src/Sitewards/DBCompare/Question/WorkerQuestion.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Sitewards\DBCompare\Question;
+
+use Sitewards\DBCompare\Worker\Item\StoreConfigWorker;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+
+class WorkerQuestion
+{
+    /** @var QuestionHelper */
+    private $oQuestionHelper;
+
+    /**
+     * @param QuestionHelper $oQuestionHelper
+     */
+    public function __construct(
+        QuestionHelper $oQuestionHelper
+    ) {
+        $this->oQuestionHelper = $oQuestionHelper;
+    }
+
+    /**
+     * @param InputInterface $oInput
+     * @param OutputInterface $oOutput
+     * @return string
+     * @throws \Symfony\Component\Console\Exception\RuntimeException
+     */
+    public function getMergeWorker(
+        InputInterface $oInput,
+        OutputInterface $oOutput
+    )
+    {
+        $oItemQuestion = new ChoiceQuestion(
```

Comment:
Shouldn't we create this object when we call this method? this way we can mock the object to achieve certain behavior during tests...